### PR TITLE
Add share to Facebook Messenger

### PR DESCRIPTION
**Pull request by thisJJ**
- Add share to facebook messenger
- Add demo facebook app id
- Add redirect url

### CODE REVIEW

File: src/MessengerShareButton.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  assert(url, 'messenger.url');
+
+  return 'http://www.facebook.com/dialog/send' + objectToGetParams({
+    app_id: appId,
+    link: url,
+    redirect_uri: redirectUri,
+  });
+}
+
+const MessengerShareButton = createShareButton('messenger', messengerLink, props => ({
```
Comment (siriwut) [inline src/MessengerShareButton.js]: `MessengerShareButton` Component couldn't understand, which social media messenger you want to share

**Comment by siriwut** [comment]
why package.lock was changed?

File: src/MessengerShareButton.js (hunk 1 above)
Comment (thisJJ) [reply src/MessengerShareButton.js]: I will change to ```FacebookMessengerShareButton```, thank for comment.

**Comment by nygardk** [comment]
Thank you for your contribution! This feature has been implemented in `v4.0.0`.

**Pull request closed without merging**